### ISSUE TITLE
Add support for XGBoost regressors

### ISSUE THREAD

**Issue by pobonomo** [Gurobi/gurobi-machinelearning#177]
XGBoost is popular for gradient boosting trees. It should be relatively easy to support at least the basic regression model.

**Comment by pobonomo** [comment]
It seems to work but its missing:
 - [ ] more tests
 - [ ] documenting
 - [ ] example?
 - [ ] should we directly take the sklearn wrapper too?
 - [ ] error out if the booster is not a GBT

**Comment by pobonomo** [comment]
Was merged and added in v1.3.0
